code/__HELPERS/lists.dm

```diff
@@ -116,8 +116,7 @@
 	else
 		result = first - second
 
-	if(result.len)
-		return result
+	return result

```

Comment:
Why we'd want to return an empty difference list confuses me, but if it keeps stuff working

Guess I should take a better look at camera code.